jobs/ci-kubernetes-e2e-gce-large-manual-up.env

```diff
@@ -0,0 +1,23 @@
+### provider-env
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### cluster-env
+NUM_NODES=1000
+NODE_SIZE=n1-standard-1
+NODE_DISK_SIZE=50GB
+CLUSTER_IP_RANGE=10.160.0.0/14
+LOGROTATE_MAX_SIZE=5G
+
+### kubernetes-env
+KUBE_OS_DISTRIBUTION=gci
+KUBE_ENABLE_CLUSTER_MONITORING=none
```

Comment:
We can, but this config is something changeable by the guy running it. The plan is that this job should be triggerable from his computer locally (instead of going to jenkins and kicking it, which also means two people can't run the same job at once).

For performance tests we don't usually use this, but I can enable it if you want.